doc/classes/File.xml

```diff
@@ -48,7 +49,6 @@
 			</argument>
 			<description>
 				Returns [code]true[/code] if the file exists in the given path.
-				[b]Note:[/b] Many resources types are imported (e.g. textures or sound files), and that their source asset will not be included in the exported game, as only the imported version is used (in the [code]res://.import[/code] folder). To check for the existence of such resources while taking into account the remapping to their imported location, use [method ResourceLoader.exists]. Typically, using [code]File.file_exists[/code] on an imported resource would work while you are developing in the editor (the source asset is present in [code]res://[/code], but fail when exported).
```

Comment:
You've not only moved information, but removed some, as a reader I would be interested in the implementation details why this has to be done, unless this information is available elsewhere of course, then I'm sorry.

Reply:
I don't think going over the specifics of the import process is really necessary, and makes the note too big. As long as someone knows when to use, or not use something I think that's good enough. Regardless you can find some of this information on the import process page. And I have an open PR in the docs repository expanding on what's there. https://github.com/godotengine/godot-docs/pull/3954